<?xml version='1.0' encoding='utf-8'?>
<sdf version="1.5">
    <!--Generated on demand-->
    <model name="stair_floor">
        <static>true</static>
        <link name="back">
            <pose>14.450027112651753 0.0 2.0906248237009070.0 0.0 0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 10.0 2.0</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 10.0 2.0</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="right">
            <pose>8.450027112651753 -5.0 2.0906248237009070.0 0.0 1.5707963267948966</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 12.0 2.0</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 12.0 2.0</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="left">
            <pose>8.450027112651753 5.0 2.0906248237009070.0 0.0 1.5707963267948966</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 12.0 2.0</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 12.0 2.0</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="front_right">
            <pose>2.450027112651753 -3.0 2.0906248237009070.0 0.0 0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 4.0 2.0</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 4.0 2.0</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="front_left">
            <pose>2.450027112651753 3.0 2.0906248237009070.0 0.0 0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 4.0 2.0</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 4.0 2.0</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="side_left">
            <pose>1.2250135563258766 1.0 1.54531241185045350.0 0.0 1.5707963267948966</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 2.450027112651753 3.090624823700907</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 2.450027112651753 3.090624823700907</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="side_right">
            <pose>1.2250135563258766 -1.0 1.54531241185045350.0 0.0 1.5707963267948966</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.01 2.450027112651753 3.090624823700907</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <box>
                        <size>0.01 2.450027112651753 3.090624823700907</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="floor">
            <pose>8.450027112651753 0.0 1.08562482370090720.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>12.0 10.0 0.005</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>105 105 105 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>12.0 10.0 0.005</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_0">
            <pose>0.17500193661798238 0.0 0.077901773121493360.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_1">
            <pose>0.5250058098539472 0.0 0.233705319364480080.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_2">
            <pose>0.8750096830899119 0.0 0.389508865607466840.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_3">
            <pose>1.2250135563258768 0.0 0.54531241185045350.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_4">
            <pose>1.5750174295618415 0.0 0.70111595809344030.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_5">
            <pose>1.9250213027978063 0.0 0.8569195043364270.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
        <link name="step_6">
            <pose>2.275025176033771 0.0 1.01272305057941360.0 0.0 0.0</pose>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <material>
                    <ambient>112 128 144 1</ambient>
                </material>
                <geometry>
                    <box>
                        <size>0.35000387323596477 2.0 0.15580354624298673</size>
                    </box>
                </geometry>
            </visual>
        </link>
    </model>
</sdf>
